app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AdminController < ApplicationController
+  def index
+    @top_customers = Customer.top_five_by_successful_transactions
```

Comment:
Nice use of this method in the controller - Caroline and I could implement into our too